### ISSUE TITLE
Enabling publishing as iOS, Android, etc. app

### ISSUE THREAD

**Issue by RobDolinMS** [jmoenig/Snap--Build-Your-Own-Blocks#1363]
Is there any effort to enable/encourage/support publishing projects as "apps" for iOS, Android, Windows 10/WinPhone, Kindle Fire, etc.?

(I see there is a compiler for desktop apps: http://snapp.citilab.eu/ )

With advancements in "Progressive Web App" technologies, I'm thinking this might be relatively straightforward.  (Ignoring app store legal issues.)

Thanks--
--Rob
(Past instructor of Intro to C.S. at Franklin H.S. in Seattle, WA through the http://www.tealsk12.org/ program)

**Comment by bromagosa** [comment]
Indeed, extending Snapp! to also build Cordova-based mobile apps is totally feasible and not very difficult. It does take quite a lot of time though, a precious resource we all are short of :(

I'm adding this to my ToDo though.

**Comment by brianharvey** [comment]
We can do the equivalent of making mobile apps by making desktop shortcuts to URLs. So when you click on it, you're really running Safari or Chrome, but it still acts like an app.  We teach this to our students early on.

**Comment by bromagosa** [comment]
Brian, this is a cool trick indeed, but it has the disadvantage of needing a connection to the Internet at all times and consuming data at launch time. What the Snapp! approach would provide is an offline application that, who knows, could even be pushed to the Play Store if a Snap! programmer wants to distribute it.

**Comment by brianharvey** [comment]
Oh, absolutely, I'm not saying we shouldn't have that, especially because it would let us use phone data like App Inventor.  All I'm saying is that meanwhile, the OP can have his students making apps right now.
